# Microbial Ecology Workshop
Files, instructions and presentation during a presentation and workshop I gave on microbial ecology during the Annual Fall Workshops in Limnology at the Interuniversity Research Group for Limnology Research (GRIL) - EcoLac.
## About
Every year, undergrads and graduate students with a diverse background in aquatic science and limnology attend the Annual Fall Workshop organised by GRIL-EcoLac. This year, we gave a conference and workshop on the theme of microbial ecology.
This workshop is aimed for a beginner's level.
## Conference Abstract
Microbes are present everywhere and they are crucial to the dynamics of global biogeochemical cycles. In freshwater ecosystems, they play important roles by consuming and producing a myriad of substances including compounds that will fuel food webs, nutrients, and greenhouse gases. Therefore, identifying microbes in the environment as well as determining what regulates their activity is essential to our understanding of ecosystem functioning. But, even though they are very abundant and ubiquitous, their identification is not trivial because of their microscopic size. In this workshop we will explore the different roles of microorganisms in freshwater ecosystems and present some molecular and genomic methods used to detect their presence or activity in lakes.
## Workshop Abstract
In this workshop, participants will explore microbial diversity and metabolism. Students will be given the option of three scenarios to answer questions regarding metabolic, genomic diversity and biogeographic patterns. The participants will learn about core concepts and terminology in bioinformatics and genomics, file formats, how to retrieve sequences from online databases, have a critical understanding on curated data. They will apply this knowledge by creating a phylogenetic tree, visualizing and creating a publication ready tree using free softwares. Finally, they will answer a few questions about the patterns observed in the tree and how it applies to microbial diversity and metabolism. They will share their observations and discuss with the rest of the group at the end of the activity.
## Tutorial
## Before getting started
In this tutorial, we will use these softwares to create and visualize a 16S rRNA gene phylogeny. We also recommend knowing where and how to access the Terminal and a text editor, we will give explanations for that soon. 
1. MEGA 7 http://www.megasoftware.net/
2. FigTree http://tree.bio.ed.ac.uk/software/figtree/
## Download required files:
There are three folders in this repository called "Scenario_1", "Scenario_2" and "Scenario_3". In each file you will find 4 fasta files: 1 containing the reference sequences, 1 with sequences from the outgroup and 2 with sequences of the microbial communities you will study in each scenario.
1. Scenario 1:
Samples were taken from a temperate lake (Lake Croche, in Canada) and a tropical lake (Lake Carioca, in Brazil, data provided by Marcelo √Åvila) at the same season. We will explore the similarity and differences in microbial community composition.
2. Scenario 2:
Samples of anoxygenic and oxygenic photosynthetic bacteria were taken from public databases. We will compare how phylogenetic different these species can be.
3. Scenario 3:
Samples were taken from two lakes of different trophic status: Lake Croche (Canada), an oligotrophic lake, and Lake Erie (USA), a large deep lake under cyanobacterial bloom. We will explore the similarity and differences in microbial community composition.
## Get ready to build the phylogenetic tree in MEGA
After you have chosen which scenario you want to study, open MEGA 7 in your computer and follow the instructions present in the tutorial (PartI_usingMEGA_ecolac). You can use the English (EN) or French (FR) version.
## Edit the phylogenetic tree in FigTree
After you have finished the tutorial Part I you can save the tree as a Newick file (.nwk) and edit it followinf the tutorial Part II (PartII_VisualizingTree_FigTree_Tutorial)
